search/search_repository.go

```diff
@@ -402,32 +405,34 @@ func (q Query) determineLiteralType(key string, val string) criteria.Expression
 	}
 }
 
-// handleWitGroup Here we handle the "$WITGROUP" query parameter which we translate from a
-// simple
+// handleWitGroup Here we handle the "$WITGROUP" and "typegroup.name" query
+// parameter which we translate from a simple
 //
-// "$WITGROUP = y"
+// "$WITGROUP = y" or "typegroup.name = y"
 //
 // expression into an
 //
 // "Type in (y1, y2, y3, ... ,yn)"
 //
-// expression where yi represents the i-th work item type associated with
-// the work item type group y.
+// expression where yi represents the i-th work item type associated with the
+// work item type group y.
 func handleWitGroup(q Query, expArr *[]criteria.Expression) error {
-	if q.Name != WITGROUP {
+	if q.Name != WITGROUP && q.Name != "typegroup.name" {
```

Comment:
Use variable?

Reply:
@aslakknutsen yes, thank you. Fixed in 0472013